src/rateEngine/__tests__/LoadProfileScaler.test.ts

```diff
@@ -58,4 +58,15 @@ describe('LoadProfileScaler', () => {
       })
     });
   });
+
+  describe('toMonthlyKwh', () => {
+    it('returns a LoadProfile scaled to the monthly load', () => {
+      const monthlyLoad = times(12, (i) => ((i + 1) * 100));
+      const scaledLoadProfile = loadProfileScaler.toMonthlyKwh(monthlyLoad);
+
+      scaledLoadProfile.sumByMonth().forEach((monthlyTotal, idx) => {
+        expect(monthlyTotal).toBeCloseTo(monthlyLoad[idx]);
```

Comment:
float math makes this end up like

```
[
  100,
  199.99999990,
  300.00000003,
  // etc
]
```

i'm assuming that's fine

Reply:
That sounds like a @euggra question

Reply:
As long as we're using `multiplyDecimals()` in the scaling math, I think yeah this is ok.